src/pages/resources.tsx

```diff
@@ -0,0 +1,176 @@
+    downloads: [
+      {
+        fileName: "TradeTrust Tech Webinar 5.pdf",
+        path: "/static/images/webinar/tradetrust-tech-webinar-5.pdf",
+      },
+    ],
+  },
+  {
+    title: "Identity Resolution Services",
+    youtubeEmbedCode: "3S1QjgPXn54",
+    tag: "Technical",
+    description:
+      "Identity Resolution in TradeTrust involves the ability to map non-readable identification of issuers to a readable form. This segment will cover the methods and the steps on building APIs for identity resolution.",
+    downloads: [
+      {
+        fileName: "TradeTrust Tech Webinar 6.pdf",
+        path: "/static/images/webinar/tradetrust-tech-webinar-6.pdf",
+      },
+    ],
+  },
+];
+
+export const ResourcesPage = () => (
+  <ResourcePageStyle>
+    <Helmet>
+      <meta
+        property="description"
+        content="This series of tech talks is organised by the Infocomm Media Development Authority of Singapore (IMDA) and GovTech Singapore. It comprises six webinars and aims to provide professionals with knowledge on TradeTrust as a digital utility for cross border trade."
+      />
+      <meta
+        property="og:description"
+        content="This series of tech talks is organised by the Infocomm Media Development Authority of Singapore (IMDA) and GovTech Singapore. It comprises six webinars and aims to provide professionals with knowledge on TradeTrust as a digital utility for cross border trade."
+      />
+      <meta property="og:title" content="TradeTrust - An easy way to check and verify your documents" />
+      <meta property="og:url" content={`${window.location.origin}/resources`} />
+      <title>TradeTrust - Resources</title>
+    </Helmet>
+
+    <div className="bg-blue-lighter">
+      <div className="container-custom py-5">
+        <div className="row">
+          <div className="col-12">
+            <h1>Resources</h1>
+          </div>
+        </div>
+        <div className="row py-3">
+          <div className="col-12 col-md-12 col-lg-4 order-lg-2">
+            <div className="side-links">
+              {sideLinks.map((resourceLink, index) => (
+                <ResourcesLink
+                  key={index}
+                  title={resourceLink.title}
+                  type={resourceLink.type}
+                  details={resourceLink.details}
+                />
+              ))}
+            </div>
+          </div>
+          <div className="col-12 col-md-12 col-lg-8 order-lg-1">
+            {resources.map((resource, index) => (
+              <ResourcesCard details={resource} key={index} />
+            ))}
+          </div>
+        </div>
+      </div>
+    </div>
+  </ResourcePageStyle>
+);
+
+const ResourcePageStyle = styled.div`
+  .side-links {
+    @media (min-width: ${vars.lg}) {
+      position: sticky;
+      top: 10px;
+    }
+  }
+`;
```

Comment:
not necessary to wrap it as the most parent, u can define it as say `Aside`, then use it directly in the column, line 148

Reply:
hmm, but i was thinking of in case in future we need to add something else, then we can just add inside.

Reply:
ic, maybe exclude wrapping the helmet tag then?

Reply:
ok understand